docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -37,11 +37,11 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
     [ ROW FORMAT row_format ]
     [ STORED AS file_format ]
     [ LOCATION path ]
-    [ TBLPROPERTIES ( key1=val1, key2=val2, ... ) ]
+    [ TBLPROPERTIES ( key1 [=] val1, key2 [=] val2, ... ) ]
     [ AS select_statement ]
 
 row_format:    
-    : SERDE serde_class [ WITH SERDEPROPERTIES (k1=v1, k2=v2, ... ) ]
+    : SERDE serde_class [ WITH SERDEPROPERTIES (key1 [=] val1, key2 [=] val2, ...)  ]
```

Comment:
you accidentally added an extra space in between of `)` and `]`?

Reply:
Thanks! I removed it.